src/app/components/ListFilter/ListFilter.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class ListFilter extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      withJS: false,
+      submitValue: '2018-01-01',
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  componentDidMount() {
+    this.setState({
```

Comment:
There's an `action` that does the same thing at the top `Application` level. That prop `isJsEnabled` is passed down to all the children components so use that instead of setting the state for in this component.

Reply:
OK we can use that then!